PicoPagesList.php

```diff
@@ -55,8 +66,30 @@ public function onPagesLoaded(
      */
     public function onPageRendering(Twig_Environment &$twig, array &$twigVariables, &$templateName)
     {
-        $twigVariables['PagesList'] = $this;
-        $twigVariables['pages_list'] = $this->html(); // backward compatibility
+        $twig->addGlobal('nested_pages', $this->items);
+        $twig->addFilter(new Twig_SimpleFilter('navigation', function($pages) {
+            return $this->output($pages);
+        }));
+        $twig->addFilter(new Twig_SimpleFilter('exclude', function($pages, $paths) {
+            $paths = self::pathsParam($paths);
```

Comment:
I was about to, but variadic filters where added in Twig 1.19. The version of Pico I have (should be the last) uses 1.18. Did I miss something ?
I hesitated much to add that comma-separated feature, with variadic or not. I finally decided that even if It's not much complex for the user to do `only(['foo', 'bar'])` instead of only('foo,bar')`, adding it as an additional markup may be found useful.

Reply:
Ha, amazing, I'll use variadic then. And drop the comma-separated.